lib/riax_kv.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Riax.KV do
+  @moduledoc """
+  Riax Key-Value API
```

Comment:
ok now we can talk a bit more about this. is the idea here like " we know 90% you'll want to do a key-value with this, so we give you the implementation"? perhaps this would be more like tutorial/example type of code instead of adding it to the library?

Reply:
Answering your second question: Yes, that's the idea here.
Riax module will be in the final library.
Riax.KV will be used as example/tutorial in the docs, eventually I'll remove this module from the code.